starknet-accounts/tests/single_owner_account.rs

```diff
@@ -31,6 +31,7 @@ fn create_sequencer_client() -> SequencerGatewayProvider {
 }
 
 fn create_jsonrpc_client() -> JsonRpcClient<HttpTransport> {
+    #[allow(clippy::or_fun_call)]
```

Comment:
Why can't we apply the clippy suggestion here?

Reply:
Why is this marked resolved?

Reply:
Saw the other comment. While I agree that it doesn't matter since it's just test code, adding an `allow` line here would actually be more verbose than just changing into `unwrap_or_else`. So I'd actually lean towards doing that instead of adding the line.